src/Standards/Squiz/Tests/WhiteSpace/OperatorSpacingUnitTest.inc

```diff
@@ -129,11 +129,11 @@ $a = $x?$y:$z;
 $a = $x ? $y : $z;
 
 $y = 1
-   + 2  
+   + 2
    - 3;
 
-$y = 1 + 
-     2 - 
+$y = 1 +
+     2 -
```

Comment:
Please revert the whitespace changes here. IIRC, the trailing spaces are testing a specific situation with the fixer.

Reply:
Sure, sorry, Phpstorm was too proactive :)